src/sentry/web/frontend/release_webhook.py

```diff
@@ -80,7 +80,12 @@ def _handle_builtin(self, request, project):
         )
 
     def post(self, request, plugin_id, project_id, signature):
-        project = Project.objects.get_from_cache(id=project_id)
+        try:
+            project = Project.objects.get_from_cache(id=project_id)
+        except Project.DoesNotExist:
+            logger.warn(
```

Comment:
I'd log this in a more structured way, something like:

```python
logger.warn('release.invalid-project', extra={'project_id': project_id})
```

Now, I know none of the other logging in this file is like this, but would probably be worth a pass of all loggers here to clean it up and unify. This can be done as a follow up.

Reply:
👍  will clean all of these up in a follow up